lib/widget/roles_selection.dart

```diff
@@ -0,0 +1,211 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_redux/flutter_redux.dart';
+import 'package:heist/app_localizations.dart';
+import 'package:heist/colors.dart';
+import 'package:heist/middleware/room_middleware.dart';
+import 'package:heist/role.dart';
+import 'package:heist/selectors/selectors.dart';
+import 'package:heist/state.dart';
+import 'package:heist/widget/common.dart';
+import 'package:redux/redux.dart';
+
+class RolesSelection extends StatefulWidget {
+  final Store<GameModel> _store;
+
+  RolesSelection(this._store);
+
+  @override
+  State<StatefulWidget> createState() => RolesSelectionState(_store);
+}
+
+class RolesSelectionState extends State<RolesSelection> {
+  final Store<GameModel> _store;
+
+  RolesSelectionState(this._store);
+
+  @override
+  Widget build(BuildContext context) =>
+      new StoreConnector<GameModel, RolesSelectionModel>(
+          converter: (store) => new RolesSelectionModel._(
+              store.state.room.numPlayers,
```

Comment:
Although the state has public member variables at the moment, I have been accessing them through selectors everywhere, e.g. `getRoom(store.state)`. We should probably make them private.